### PR TITLE
Add refresh jwt route + Fix scraper

### PR DESCRIPTION
**Pull request by NoahHakansson**
Adding a route that returns a new fresh valid JWT if the caller has a currently valid JWT.

Some improvments to how scraping and sending notifications for news is handled.

### CODE REVIEW

File: backend/api/src/DB/exams_test.go [hunk 1]
```diff
@@ -22,25 +22,29 @@ func TestDatabaseExam(t *testing.T) {
 func TestCreateExam1(t *testing.T) {
 	_, err := helpers.FixtureWrapCreate(t, &helpers.TestExam)
 	assert.Nil(t, err, "After fixture is run a create will be called that shall return no error")
+	_ = helpers.FixtureWrapNonCreate(t)
```
Comment (Gabriel-Ivarsson) [inline backend/api/src/DB/exams_test.go:25]: Is it neccessary to do for the code?, can be very expensive to run on all tests :/

File: backend/api/src/Routes/routes.go [hunk 2]
```diff
@@ -36,18 +36,19 @@ func SetupRoutes(gormObj *gorm.DB) (*gin.Engine, error) {
...
 		auth.GET("/api/auth-hello", hello)
 		auth.GET("/api/list-user-exams", listUserExams)
+		auth.POST("/api/refresh-jwt", refreshJWT)
 		auth.POST("/api/register-exam", registerToExam)
 		auth.POST("/api/unregister-exam", unregisterFromExam)
 		auth.POST("/api/add-user-expo-push-token", addUserExpoPushToken)
 		adminAuth := auth.Group("/", adminMiddleware)
 		{
 			adminAuth.GET("/api/list-exam-users", listExamUsers)
+			adminAuth.GET("/api/list-due-exams", listDueExams)
```
Comment (Gabriel-Ivarsson) [inline backend/api/src/Routes/routes.go:51]: Why should due exams be put into admin protected routes?

**Review by Gabriel-Ivarsson — CHANGES_REQUESTED** [review]
Questions from reading through changed files:
- Why is testing code changed inside of a branch called "Add refressh jwt route + fix scraper"
- Why wasn't the test-developer (me) noted of changes before they were developed on a branch
- As of now these changes are not necessary for the test enviorment to run, and might not fix parallel issues, however I do not know
- Why was listDueExams moved to admin authenticated parts of routes

What was good:
- Otherwise quality was good of code i would say, nice job with scraper, and the new route.

File: backend/api/src/Routes/routes.go (hunk 2 above)
Comment (NoahHakansson) [reply backend/api/src/Routes/routes.go:51]: It is only used as an API Route for testing purposes. Could even be removed. But just put it in admin routes instead.

**Comment by NoahHakansson** [comment]
Wrote on discord about the change and why I added the clean up after each test. It leaves the database in a non-clean state after tests are run otherwise. Which might be unwanted especially in a local dev environment.

**Review by Gabriel-Ivarsson — APPROVED** [review]
After talking with developer we found that changes were relevant to codebase

**Pull request merged**